source/sass/buyers-guide/views/category.scss

```diff
@@ -48,28 +42,53 @@
         padding-right: 3%;
       }
 
+      .seal-of-approval {
+        position: absolute;
+        z-index: 2;
+        top: $padding-y;
+        left: $padding-x;
+        width: 40px;
+        height: auto;
+
+        @media (max-width: $bp-md) {
+          width: 20px;
+          height: auto;
+        }
+      }
+
       a {
         text-decoration: none;
         color: inherit;
       }
 
       .category-image-container {
-        margin-bottom: 20px;
+        position: relative;
         overflow: hidden;
+        background: #f4f4f4;
+        margin-bottom: 12px;
+        padding: 10px;
+
+        @media (min-width: $bp-xl) {
+          padding: 20px;
+        }
       }
 
       img {
         display: block;
         width: 100%;
         height: auto;
+
+        @media (max-width: $bp-md) {
+          transform: scale(1.25);
+        }
       }
 
       .category-item-label {
```

Comment:
Ummm... I couldn't seem to find any element with class name `category-item-label` on the category page. Is this old code to be deleted?

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/2896608/49191435-5df0b700-f32b-11e8-8b5c-6164c65049f4.png">

Reply:
hm, it might very well be. We should probably figure out if there's something we can do around dead rule detection etc.

Reply:
This was a hold over from when we weren't using `h*-heading` instead, so I'll remove it + the header class which is also unused now